stripe/res/layout/activity_payment_sheet.xml

```diff
@@ -48,7 +48,7 @@
                 android:layout_height="wrap_content"
                 android:visibility="gone"
                 android:layout_marginVertical="2dp"
-                android:layout_marginHorizontal="10dp"
+                android:layout_marginHorizontal="23dp"
```

Comment:
Looks like this is slightly too much margin. It should be lined up with the "Country or region" header.